src/credentials/FabricTable.cpp

```diff
@@ -219,19 +222,10 @@ CHIP_ERROR FabricInfo::LoadFromStorage(PersistentStorageDelegate * storage)
 
         if (mOperationalKey == nullptr)
         {
-#ifdef ENABLE_HSM_CASE_OPS_KEY
-            mOperationalKey = chip::Platform::New<P256KeypairHSM>();
-            mOperationalKey->SetKeyId(CASE_OPS_KEY);
-#else
-            mOperationalKey = chip::Platform::New<P256Keypair>();
-#endif
+            mOperationalKey = chip::Crypto::GetP256KeypairBuilder()->BuildP256KeyPairForOperationalKey(mFabricId);
```

Comment:
This should be FabricIndex, not fabricID. FabricID are ambiguous across roots of trust.

Reply:
Updated.